### PR TITLE
Release notes for API Designer 2.3.0 and Mocking Service 1.1.12

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +16,46 @@ endif::[]
...
+/channels/{channel-code}/digital-users/{digital-user-id}
+----
+// SE-8999
+* The mocking service returned a 400 error for a PATCH request that met these two conditions:
+** The request had a JSON body of a type that was defined in a library.
+** The library defined the type with an `!include` tag.
+// SE-8790
+* In some cases, validation of required content was not working for request bodies. Instead, payloads were accepted from the API Console without any validation.
+// SE-8745
+* In some cases, the Mocking Service accepted requests that did not contain required parameters.
```
Comment (dukesphere) [inline modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: "Mocking Service" or "mocking service"?

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc [hunk 2]
```diff
@@ -13,6 +16,46 @@ endif::[]
...
+== API Mocking Service 2.0.0
+
+// SE-9588
+* The mocking service returned a 404 error instead of the correct response if a third-level resource used a hyphen in its name. For example, this call `digital-users` is the third-level resource would cause the issue:
++
+----
+/channels/{channel-code}/digital-users/{digital-user-id}
+----
+// SE-8999
+* The mocking service returned a 400 error for a PATCH request that met these two conditions:
```
Comment (dukesphere) [inline modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: "mocking service" here is "Mocking Service" in some other places

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc [hunk 3]
```diff
@@ -13,6 +16,46 @@ endif::[]
...
+
+
+
+== API Mocking Service 1.1.11
+
+August 21, 2018
+
+This release fixes the following issue:
+
+* The mocking service did not return messages for the 400 response in with the correct styling.
```
Comment (dukesphere) [inline modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: lower case "mocking service" here

Comment (dukesphere) [reply modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: "in with" -> "with" ?

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc [hunk 4]
```diff
@@ -13,6 +16,46 @@ endif::[]
...
+
+
+== API Mocking Service 1.1.10
+
+June 30, 2018
+
+This release fixes the following issue:
+
+* TLS 1.0 and 1.1 were still supported.
```
Comment (dukesphere) [inline modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: Does this mean that they are no longer supported? Or they were supported but weren't supposed to be? Maybe it's me...

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 5]
```diff
@@ -5,12 +5,59 @@ endif::[]
...
 * <<2-2-5,2.2.5>>
 
+== 2.3.0
+*November 1, 2018*
+
+This release contains an enhancement and fixes a number of issues.
+
+=== Enhancement
+
+The API editor is now using the AMF parser. This parser checks for full conformance with the RAML specification (either 0.8 or 1.0) that is being used for an API specification. If you reopen in the API editor any RAML API specifications that have already been published, you might find that there are instances of invalid RAML that the previous parser did not flag. For more information about what to do in such cases, see xref:design-center::design-modify-raml-specs-conform.adoc[Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0].
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Does this file exist? xref:design-center::design-modify-raml-specs-conform.adoc[Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0].

Comment (dukesphere) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: I don't see it in the `latest` branch of `docs-design-center`, and it's showing up as a failed link in my local build. I also don't see it on our prod site.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 6]
```diff
@@ -5,12 +5,59 @@ endif::[]
...
+* The API console pane of the API editor was sometimes not being refreshed as different types were being selected. The same problem could occur as different responses were selected.
+* There were a few formatting problems in the information displayed in the API console pane.
+// SE-8242
+* If a RAML API specification contained a DataType that extended another DataType, the information shown in the API console pane about the extending DataType could sometimes be incorrect.
+// SE-8323
+* The API editor could take too long to resolve one or more DataType resources. During that time, the API console pane was blank.
+// SE-8424
+* When a RAML extension extended a file that used a library that was itself defined in a fragment, and the library included a trait with the `!include` tag, the API editor was unable to locate the trait.
+// SE-8464
+* If a RAML API fragment was defined multiple properties as `<propertyName>: string | nil`, the API console pane did not display any information, or did so only after a long delay, after the API specification that referenced the fragment was opened.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: "was defined" -> "defined"

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 7]
```diff
@@ -5,12 +5,59 @@ endif::[]
...
+// SE-9121
+* The API editor was requiring the media type to be declared for message bodies, even though the RAML specifications state that `mediaType` is optional for `body` nodes.
+// SE-9269
+* The Mocking Service was not presenting a choice of authentication methods when a basic securityScheme was being referenced from a library RAML fragment.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: securityScheme -> `securityScheme`

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc (hunk 5 above)
Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: It'll be published on the same day that these release notes are published.

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: Fixed. Changed to lowercase.

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: Fixed.
Kept in lowercase.
Removed "with".

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc (hunk 4 above)
Comment (rddheath) [reply modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: Only 1.2 is supported now. Fixed.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc (hunk 6 above)
Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Removed "was".

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc (hunk 7 above)
Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Fixed.

File: modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/api-mocking-service/api-mocking-service-release-notes.adoc]: Changed all to lowercase.

**Pull request merged**